pycodestyle.py

```diff
@@ -133,7 +133,9 @@ def lru_cache(maxsize=128):  # noqa as it's a fake implementation.
 RAISE_COMMA_REGEX = re.compile(r'raise\s+\w+\s*,')
 RERAISE_COMMA_REGEX = re.compile(r'raise\s+\w+\s*,.*,\s*\w+\s*$')
 ERRORCODE_REGEX = re.compile(r'\b[A-Z]\d{3}\b')
-DOCSTRING_REGEX = re.compile(r'u?r?["\']')
+DOCSTRING_REGEX = re.compile('(u|r|ur|ru' +
+                             ('|fr|rf' if sys.version_info >= (3, 6) else '') +
```

Comment:
docstrings cannot be prefixed with `f`:

```pycon
>>> class C:
...     f'hi'
... 
>>> C.__doc__
>>> 
```

Reply:
Thanks, I updated the PR and description to drop the change regarding `f` and `F`.